docs/source/java/vector.rst

```diff
@@ -118,6 +118,28 @@ Some points to note about the steps above:
   no longer used, to avoid resource leak. To make sure of this, it is recommended to place vector related operations
   into a try-with-resources block.
 
+* For fixed width vectors (e.g. IntVector), we can set values at different indices in arbitrary orders.
+  For variable width vectors (e.g. VarCharVector), however, we must set values in non-decreasing order of the
+  indices. Otherwise, the values after the set positiion will become invalid. For example, suppose we use the
+  following statements to populate a variable width vector:
+
+.. code-block:: Java
+
+    VarCharVector vector = new VarCharVector("vector", allocator)) {
```

Comment:
Typo here - was this meant to be a try-with-resources or a semicolon?

Reply:
Typo fixed. Thanks a lot for your careful review.